test/e2e/resources.cy.js

```diff
@@ -1,63 +1,74 @@
-import Course from '../../../src/course/en/course.json'
-
 describe('Resources', () => {
-    const checkDrawerLength = (count) => {
-        cy.get('.drawer__item').not('.u-display-none').should('have.length', count)
-      }
+  const ITEM_TYPES = [
+    'document',
+    'link',
+    'media'
+  ];
```

Comment:
Not all courses will have these three types. Perhaps derive the ITEM_TYPES from the config?